benchexec/tools/nitwit.py

```diff
@@ -50,24 +43,24 @@ def name(self):
     def cmdline(self, executable, options, task, rlimits):
         if task.property_file:
             options = options + ["-p", task.property_file]
-        return [executable] + options + list(task.input_files_or_identifier)
+        return [executable] + options + task.single_input_file
 
     def determine_result(self, run):
         """
         See README.md at https://github.com/moves-rwth/nitwit-validator for information
         about result codes.
         @return: status of validator after executing a run
         """
-        if run.exit_code.signal is None and (run.exit_code.value == 0 or run.exit_code.value == 245):
+        if run.exit_code.value in [0, 245]:
             status = result.RESULT_FALSE_REACH
-        elif run.exit_code.value is None or run.exit_code.value in [-9, 9]:
+        elif (run.exit_code.signal == 9) or run.exit_code.value == 9:
             status = "TIMEOUT"
         elif run.exit_code.value in [4, 5, 241, 242, 243, 250]:
             status = result.RESULT_UNKNOWN
         else:
             status = result.RESULT_ERROR
 
-        if run.output.any_line_contains("Out of memory") or (run.exit_code.value == 251):
+        if ("out of memory!".lower() in (line.lower() for line in run.output)) or (run.exit_code.value == 251):
```

Comment:
Please reformat with `black` or apply the [diff](https://travis-ci.org/github/sosy-lab/benchexec/jobs/735977162#L945) from CI manually, this line is too long.

Reply:
Done.